### ISSUE TITLE
[XHTML Friends Network] Accept `rel="parent"` value

### ISSUE THREAD

**Issue by stonio** [validator/validator#2134]
### Summary

https://validator.w3.org/nu/#textarea report a misleading INFO message:
>Potentially bad value `parent` for attribute `rel` on element [`a`](https://html.spec.whatwg.org/#the-a-element) Typo for `payment`?

Whereas  `rel="parent"` is defined in <https://gmpg.org/xfn/11>.

Other XHTML Friends Network `rel` values are supported though.

### URL for document that reproduces the problem

n/a

### HTML/CSS/SVG snippet that reproduces the problem

```html
<!DOCTYPE html>
<html lang="en">
  <head>
    <title>Test XFN</title>
  </head>
  <body>
    <a href="#contact" rel="contact">contact</a>
    <a href="#acquaintance" rel="acquaintance">acquaintance</a>
    <a href="#friend" rel="friend">friend</a>
    <a href="#met" rel="met">met</a>
    <a href="#co-worker" rel="co-worker">co-worker</a>
    <a href="#colleague" rel="colleague">colleague</a>
    <a href="#co-resident" rel="co-resident">co-resident</a>
    <a href="#neighbor" rel="neighbor">neighbor</a>
    <a href="#child" rel="child">child</a>
    <a href="#parent" rel="parent">parent</a>
    <a href="#sibling" rel="sibling">sibling</a>
    <a href="#spouse" rel="spouse">spouse</a>
    <a href="#kin" rel="kin">kin</a>
    <a href="#muse" rel="muse">muse</a>
    <a href="#crush" rel="crush">crush</a>
    <a href="#date" rel="date">date</a>
    <a href="#sweetheart" rel="sweetheart">sweetheart</a>
    <a href="#me" rel="me">me</a>
  </body>
</html>
```

### Checker error messages

```shell
Info: Potentially bad value parent for attribute rel on element a Typo for payment?
```

### Spec links/citations

https://gmpg.org/xfn/11

### Steps to reproduce

1.n/a
2.
3.

### Expected behavior

n/a

### Actual behavior

n/a

**Comment by sideshowbarker** [comment]
Thanks for taking time to report this. I’ve pushed a fix to the live checker:

https://validator.w3.org/nu/?doc=data%3Atext%2Fhtml%3Bcharset%3Dutf-8%3Bbase64%2CPCFET0NUWVBFIGh0bWw%2BCjxodG1sIGxhbmc9ImVuIj4KICA8aGVhZD4KICAgIDxtZXRhIGNoYXJzZXQ9InV0Zi04Ij4KICAgIDx0aXRsZT5UZXN0IFhGTjwvdGl0bGU%2BCiAgPC9oZWFkPgogIDxib2R5PgogICAgPGEgaHJlZj0iI2NvbnRhY3QiIHJlbD0iY29udGFjdCI%2BY29udGFjdDwvYT4KICAgIDxhIGhyZWY9IiNhY3F1YWludGFuY2UiIHJlbD0iYWNxdWFpbnRhbmNlIj5hY3F1YWludGFuY2U8L2E%2BCiAgICA8YSBocmVmPSIjZnJpZW5kIiByZWw9ImZyaWVuZCI%2BZnJpZW5kPC9hPgogICAgPGEgaHJlZj0iI21ldCIgcmVsPSJtZXQiPm1ldDwvYT4KICAgIDxhIGhyZWY9IiNjby13b3JrZXIiIHJlbD0iY28td29ya2VyIj5jby13b3JrZXI8L2E%2BCiAgICA8YSBocmVmPSIjY29sbGVhZ3VlIiByZWw9ImNvbGxlYWd1ZSI%2BY29sbGVhZ3VlPC9hPgogICAgPGEgaHJlZj0iI2NvLXJlc2lkZW50IiByZWw9ImNvLXJlc2lkZW50Ij5jby1yZXNpZGVudDwvYT4KICAgIDxhIGhyZWY9IiNuZWlnaGJvciIgcmVsPSJuZWlnaGJvciI%2BbmVpZ2hib3I8L2E%2BCiAgICA8YSBocmVmPSIjY2hpbGQiIHJlbD0iY2hpbGQiPmNoaWxkPC9hPgogICAgPGEgaHJlZj0iI3BhcmVudCIgcmVsPSJwYXJlbnQiPnBhcmVudDwvYT4KICAgIDxhIGhyZWY9IiNzaWJsaW5nIiByZWw9InNpYmxpbmciPnNpYmxpbmc8L2E%2BCiAgICA8YSBocmVmPSIjc3BvdXNlIiByZWw9InNwb3VzZSI%2Bc3BvdXNlPC9hPgogICAgPGEgaHJlZj0iI2tpbiIgcmVsPSJraW4iPmtpbjwvYT4KICAgIDxhIGhyZWY9IiNtdXNlIiByZWw9Im11c2UiPm11c2U8L2E%2BCiAgICA8YSBocmVmPSIjY3J1c2giIHJlbD0iY3J1c2giPmNydXNoPC9hPgogICAgPGEgaHJlZj0iI2RhdGUiIHJlbD0iZGF0ZSI%2BZGF0ZTwvYT4KICAgIDxhIGhyZWY9IiNzd2VldGhlYXJ0IiByZWw9InN3ZWV0aGVhcnQiPnN3ZWV0aGVhcnQ8L2E%2BCiAgICA8YSBocmVmPSIjbWUiIHJlbD0ibWUiPm1lPC9hPgogIDwvYm9keT4KPC9odG1sPgo%3D
